### DISCUSSION TITLE
CVD and replacing the term "Color Blind"

### DISCUSSION THREAD

**Discussion by Myndex** [w3c/low-vision-a11y-tf#121] [Ideas]
# CVD
## What is Color Vision Deficiency?
CVD is a condition wherein some of the "color sensing" cells of the retina are impaired or missing, resulting in difficulty discriminating between certain colors. Some variation of this condition affects an estimated 6% of the population, primarily males. However, it is important to know that the vast majority of CVD individuals can still see _some_ colors.

### Poor Choice of Words: "Color Blind"
"Color Blind" is not accurate in terms of what CVD to those with some variation of this condition. 

#### 99.998% of those with CVD are not color _blind_
They only have a _reduced_ color discrimination. Even the most profound deuteranope or protanope can still see hundreds of thousands of colors (far shy of the many millions of normal vision, but not devoid of color perception).

#### Is there an "actual" color "blind"?
Only the very rare achromatopsia types are monochromatic, and that very small group often has other more serious vision problems that require assistive technologies, regardless of their lack of color discrimination. (For instance, blue cone monochromacy is co-morbid with a low vision of 20/70 and severe photophobia. There is even some discussion regarding taking blue cone monochromacy out of the CVD classification all together.)

#### What is it really like?
You can see various forms of CVD based on clinical research at [https://www.myndex.com/CVD/ the **Myndex CVD Simulator page**.] The protanopia, deuteranopia, and tritanopia  simulations are based on the Brettel models which use the cone response functions and are generally accepted as clinically accurate. The Blue Cone Monochromacy I created using the Brettel concepts, however the current blue cone simulator does nto also simulate the central field loss or low vision aspects of BCM.

Like most things in perception, CVD is not an absolute, but a wide spectrum of subtle variations from a mild reduction in discrimination ability, to a complete inability to discern between reds and greens for instance (deuteranopia/protanopia). But these most common types do see blue, and related colors that stimulate the S cone (blue). 

In short, the _vast majority_ of individuals with Color Vision Deficiency have these characteristics::
- They see the opponent colors yellow/blue.
- They have difficulty distinguishing between colors that rely on differential stimulation of the M and L cones.
    - The M and L cones are often (albeit inaccurately) called the green and red cones. 
        - In fact, **both** the M and L cones have peak sensitivity in green, close to each other.
        - Both the M and L cones almost completely overlap
        - Only the L cone really extends down into deep "red"
    - The overlap means that deuteranopia (missing the M cone) sees *all* the light of normal vision, but colors from green down to red all seem the "same" hue.
- They have generally normal luminance perception
    - However, protanopia see the "red" on a sRGB monitor but as 30% to 50% darker.
    - Protanopia _can't_ see the red on a Rec2020 wide gamut monitor.
- With than in mind, contrast sensitivity and visual acuity are otherwise normal.

## A More Accurate Term is Needed
Instead of **Color Blind**, we need a new term that is not so completely incorrect (not to mention a bit ableist.) The medical terms for some disabilities are changed every so often for similar reasons, It should not be too difficult to change "colorblind" to a better and more correct/inclusive term with the appropriate promotion.

### _Candidate Term:_
- **Color Limited** used as:
    - "he's color limited, sometimes (but inaccurately) referred to as 'color blind'"
    - "They have color limited vision"
    - "Some people with color limited vision may not be able to distinguish between some hues, which is some cases gives them a better ability to identify "camouflage"ª.

**_Color Limited Vision_** is neutral and descriptive of the actual condition. 

Thank you,

Andy

**Footnote:** ª camouflage being more visible due to a greater ability to discern luminance variations irrespective of hue among some CVD types. _see: https://www.nature.com/articles/news051205-1_

**Comment by shawna-slh** [comment]
+1 for replacing "color blindness" because it is a misnomer and can lead to misunderstanding

-1 for "color limited vision" because it feels like a made-up euphemism and that is very uncomfortable to me

+1 for "color vision deficiency" because it is accurate, clear, widely used, and understandable

+1 for including something like "(often called 'color blindness')" on first reference because most people will recognize 'color blindness'

+1 for briefly addressing potential misunderstandings where smooth to do so, e.g., [2.4.5 Color Vision section of Accessibility Requirements for People with Low Vision](https://www.w3.org/TR/low-vision-needs/#color-vision)

**Comment by shawna-slh** [comment]
Support for "color vision deficiency", especially among people who have it:
* https://www.oatext.com/appropriate-terminology-in-the-nomenclature-of-the-color-vision-deficiency.php
* https://www.color-blindness.com/2007/01/15/color-blindness-is-not-color-blindness/
* https://www.color-blindness.com/2007/02/09/terminology-of-color-blindness/

Examples of authoritative sources using different terms:
* [colour vision deficiency (colour blindness) &mdash; NHS, 2019](https://www.nhs.uk/conditions/colour-vision-deficiency/)
* [colour vision deficiency &mdash; RNIB, 2017](https://www.rnib.org.uk/nb-online/colour-vision-deficiency)
* [color vision deficiency &mdash; Library of Medicine](https://medlineplus.gov/genetics/condition/color-vision-deficiency/)
* [color vision deficiency &mdash; AOA](https://www.aoa.org/healthy-eyes/eye-and-vision-conditions/color-vision-deficiency)
* [color blindness &mdash; NAI NIH, 2019](https://www.nei.nih.gov/learn-about-eye-health/eye-conditions-and-diseases/color-blindness/types-color-blindness)
* [color blindness &mdash; AAO](https://www.aao.org/eye-health/diseases/what-is-color-blindness)

**Comment by Myndex** [comment]
Thank you, yes CVD is the most common clinical definition… the NIH is who we need to lobby for hte “bigger” change...

**Comment by shawna-slh** [comment]
Notes from [LVTF meeting 7 October](https://www.w3.org/2021/10/07-lvtf-minutes.html#t02)
* terminology brainstorm: **color impairment**
* some people do not consider their color deficiency a disability
* many prefer not to share broadly that they have it
* see how it is currently addressed in the [2.4.5 Color Vision section of Accessibility Requirements for People with Low Vision](https://www.w3.org/TR/low-vision-needs/#color-vision)

**Comment by shawna-slh** [comment]
Here are instances of "color blindness" in current W3C resources, and how they could be reworded to "color vision deficiencies":

**UPDATE**: _To keep this GitHub Discussion trim, the examples that were in this comment are moved to a wiki page: [Replacing ColorBlindness, 1 All to color vision deficiencies](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Replacing_ColorBlindness#1_All_to_color_vision_deficiencies)_

**Comment by shawna-slh** [comment]
Background: [EOWG discussion 8 October 2021](https://www.w3.org/2021/10/08-eo-minutes#item07)

Idea for further consideration: "**perceives color differently**"

Question: Is that too euphemism-y, e.g., too close to "differently-abled"?

Here are instances of "color blindness" in current W3C resources, and how they could be reworded to "perceive color differently" in most resources and "color vision deficiencies" in the Low Vision User Requirements doc :

**UPDATE**: _To keep this GitHub Discussion trim, the examples that were in this comment are moved to a wiki page: [Replacing ColorBlindness, 2 Some to perceives color differently](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Replacing_ColorBlindness#2_Some_to_perceives_color_differently)_

**Reply by Myndex** [reply]
HI @shawna-slh 
> Idea for further consideration: "**perceives color differently**"
> Question: Is that too euphemism-y, e.g., too close to "differently-abled"?

I waited a week before commenting...  Very Apple/Steve Jobs...  But importantly IMO not properly definitive nor descriptive. I wrote and deleted about 400 words here once or twice. To keep this short, I'll assume the foundation as read and leave the bulk deleted.

My general opinion regarding "PC"
- _To obfuscate the nature of a thing and/or twist it into something it is not, is a chilling example of newspeak, and is just as bad if not worse than the "offending" term it was intended to replace._

My interest in changing "color blind" is that it is _incorrect_. Incorrect means misunderstanding, and misunderstanding breeds prejudice. I object to the idea of changing the term to something obtuse and non-descriptive in the opposite direction.

### Terminology template
A template of a functional definition that is stable and emotionally neutral (to the extent possible) described as:
- One word that describes the topic or category, with a neutral or positive tone, and
- One word that accurately describes the extent or severity of the related condition.
- Both words must have broad, well understood definitions, not be clinical jargon, not invented for the purpose, and not co-opted from a non-related, non-clinical source.
- The complete term should not require any explanation nor qualifiers.

This was the genesis of "color limited" — color, the specific topic, and limited, descriptive of the extent. Blind is "commonly" understood as "sees nothing". Disabled is a bit stronger than limited, so is impaired, though impaired is a good candidate.

Adding "vision" or "sight" is optional as redundant (I think) since color is associated with vision. The only other common word for color I can think of is hue. Chroma is too much of a jargon term. Though there are a number of terms for a reduced capacity. Here's a mix and match chart to see if it stimulates ideas:

### Mix & Match pick one from each column
| TOPIC | CONDITION |
|-----|-----|
| Color	| Limited |
| Hue	| Reduced |
| | Disabled |
| | Impaired |
| 	| Inhibited |
| 	| Diminished |
| 	| Hindered |
| 	| Attenuated |
| 	| Hampered |
| 	| Suppressed |
| | Blocked |

## Existing Materials
> _Here are instances of "color blindness" in current W3C resources,_

Here is today's tangential commentary, having nothing to do with the term "color blind". The following paragraph is one example of the need for significant revision of certain documents.

> _Some people perceive color differently and cannot see certain colors well or at all. This is usually because of deficiencies in the cone receptors of their eyes that are responsible for color perception. This is commonly called “color blindness”, even though most people can see most colors. It is rare that a person cannot see any color at all. Globally, approximately 1 in 12 men (8%) and 1 in 200 women have color vision deficiencies. [References.] Color vision deficiencies are not classified as “low vision” or disabilities in many contexts._

This LV doc, and the supplemental guidance doc, are plagued by unqualified and duplicative use of "some", "many", "most", "usually". The text weaves in and out of passive voice and stumbles through meaningless and unsubstantiated run-on sentences like _"Color vision deficiencies are not classified as 'low vision' or disabilities in many contexts."_ The text is littered with incorrect statistics and unnecessary jargon. I previously pointed out that many of the infographics are incorrect. A lot of work needed here, as in a fresh start.

**Comment by mraccess77** [comment]
I'd support the term CVD and also @shawna-slh that color limited is potential problematic.

**Reply by Myndex** [reply]
> I'd support the term CVD and also @shawna-slh that color limited is potential problematic.

I'm not married to "color limited" — so long as the term/phrase is accurate and descriptive.

**Comment by Myndex** [comment]
Per @sdw32 's mention at the meeting regarding his preference of "impaired" as the descriptive term, and mostly favorable agreement with no objections that I recall, I'm now using **"color impaired"** in what I write, and we'll see what the response is.

**Comment by lauracarlson** [comment]
As I mentioned at our last meeting, it is good to be aware that the word "impaired" can currently be a controversial one, especially for many in the Deaf community. More than any of the other groups of individuals, those who are Deaf are much less inclined to think of their condition as an impairment. The "Hearing-impaired" section of [National Association of the Deaf's Frequently Asked web page](http://www.nad.org/issues/american-sign-language/community-and-culture-faq) states:

> …the term "hearing-impaired" is viewed as negative.  The term focuses on what people can't do.  It establishes the standard as "hearing" and anything different as "impaired," or substandard, hindered, or damaged.  It implies that something is not as it should be and ought to be fixed if possible.  To be fair, this is probably not what people intended to convey by the term "hearing impaired…"

**Reply by Myndex** [reply]
> _As I mentioned at our last meeting, it is good to be aware that the word "impaired" can currently be a controversial one, especially for many in the Deaf community. More than any of the other groups of individuals, those who are Deaf are much less inclined to think of their condition as an impairment._ 

Hi @lauracarlson 

Thank you Laura, sorry I missed that in the meeting. It is why I was originally wanting to use "Limited" because I was concerned of the emotional weight of impaired, but for some reason I thought there was a general, if interim, "okay" feeling in the meeting.

Dang... 

Well of the others on the list, I see these other two as best candidates.

- Color Reduced

- Color Disabled

More thoughts?

**Comment by shawna-slh** [comment]
Here is an updated proposal based on above and [LVTF discussion on 21 October](https://www.w3.org/2021/10/21-lvtf-minutes.html#item01) and [EOWG discussion on 22 October](https://www.w3.org/2021/10/22-eo-minutes#item04):

> Avoid “color blind” alone, primarily because it can be misunderstood that people do not see any color.
> 1. Use “cannot distinguish between certain colors (often called “color blindness”)” in almost all W3C documents, especially referring to a person or persona.
> 2. In rare cases, when addressing the medical condition, use “color vision deficiency”. Usually do not use an acronym, because it is not widely known.<br>_The only document identified so far for this use is [Accessibility Requirements for People with Low Vision](https://www.w3.org/TR/low-vision-needs/#color-vision)_.

<hr>


1. Here are instances of "color blindness" referring to a person in current W3C resources, and how they could be reworded to "cannot distinguish between certain colors":

## [Understanding 1.4.1: Use of Color]( https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)

Change:
> Users who have color-blindness benefit when information conveyed by color is available in other visual ways.

to:
> Users who cannot distinguish between certain colors (often called “color blindness”) benefit when information conveyed by color is available in other visual ways.

## [Colors with Good Contrast](https://www.w3.org/WAI/perspective-videos/contrast/)

Change:
> People with color blindness who cannot distinguish between certain colors.

to:
> People who cannot distinguish between certain colors (often called “color blindness”).

## [User Story, Lee - Draft Update]( https://deploy-preview-113--wai-people-use-web.netlify.app/people-use-web/user-stories-four/)

Change:
> Lee, online shopper with color blindness
> Lee was born with deuteranopia and protanopia, commonly referred to as red/green color blindness. He has difficulty distinguishing among items that are red, green, orange, and brown, all of which appear kind of a murky brown.

to:
> Lee, online shopper who cannot distinguish between certain colors (color blindness)
>Lee was born with deuteranopia and protanopia, commonly referred to as red/green “color blindness”. He cannot distinguish between items that are red, green, orange, and brown, all of which appear kind of a murky brown.

<hr>

2. Here is an instance of "color blindness" referring to the medical condition in a W3C resource, and how it could be reworded to "color vision deficiencies":

## [2.4.5 Color Vision section of Accessibility Requirements for People with Low Vision](https://www.w3.org/TR/low-vision-needs/#color-vision)

Change wording at [2.4.5 Color Vision](https://www.w3.org/TR/low-vision-needs/#color-vision) to:

**2.4.5 Color Vision**

Some people cannot see certain colors well or at all, usually because of deficiencies in the cone receptors of their eyes that are responsible for color perception. This is commonly called “color blindness”, even though most people can see most colors. It is rare that a person cannot see any color at all. Globally, approximately 1 in 12 men (8%) and 1 in 200 women have color vision deficiencies. [References.] Color vision deficiencies are not classified as “low vision” or disabilities in many contexts.

Simulated examples of color vision deficiencies:
* Color wheel with 12 different colors Fig. 4 Full color perception
* Color wheel with shades of blue and yellow Fig. 5 Red-green color deficiency (deuteranopia)
* Color wheel with shades of green and red Fig. 6 Blue-yellow color deficiency (tritanopia)
* Color wheel with shades of gray Fig. 7 No color perception (rare)

User needs related to color vision deficiencies are addressed in the following section:
* 2.1 Brightness and Color

<hr>
_**Note:** We might want to edit other parts of the text. For this GitHub issue, please focus only on the terminology under consideration, and not other wordsmithing or issues._

**Reply by vmmiller** [reply]
+1

**Reply by kakinney** [reply]
+1

**Reply by lakeen** [reply]
+1

**Reply by mpalmersg** [reply]
+1

**Reply by iadawn** [reply]
+1

**Reply by jade-mc** [reply]
+1

**Reply by MicheleAWilliams-A11y** [reply]
+1

**Reply by eoncins** [reply]
+1

**Comment by sdw32** [comment]
With respect to my own writing in the public domain, and my own presentations, I'm going to start using the phrase colour impaired, and "people with colour impairments". I think this is better plain language and less of a mouthful than colour vision deficiency, and it's a more direct swap for colourblindness, and it correctly gives the indication that there is more than one different kind of colour impairment. I note the issues raised above with the term impairments, but it's better than the term blindness, so it's moving in the right direction, even if it's not perfect.

However, with regards to the supplemental guidance I think we were in agreement that colour vision deficiency is the best term, and it doesn't come up that often, so it's not too onerous to just use the full phrase. Following on from the above proposals, I'm not a huge fan of the phrase "who cannot distinguish between certain colors". How about we just use colour vision deficiencies throughout? So the condition is colour vision deficiencies, and the phrase would be "Users with colour vision deficiencies benefit when information conveyed by colour is available in other visual ways". Then, we just need to state once that colour vision deficiency is often, called colourblindness, but the blindness term is misleading etc.

**Comment by mraccess77** [comment]
Thanks @shawna-slh the proposed updates are good step forward for a tricky area to get consensus on and they are better than what we have now.

**Comment by mraccess77** [comment]
On a side note I try to stay away from terms like Vision or color Loss because it assumes people have lost vision - many people have congenital vision impairment - so it's not really a loss (although many do lose vision and vision loss can be a slow and long (or fast) progression).  I'd also strongly recommend we stay away from terms like "poor vision" or "poor color vision" as they imply something that is not appropriate.

**Comment by shawna-slh** [comment]
>   I think we were in agreement that colour vision deficiency is the best term

A few EOWG folks were uncomfortable with "deficiency" when describing a person or persona (as opposed to a medical condition). Rough notes in [22 Oct minutes](https://www.w3.org/2021/10/22-eo-minutes#item04) and [8 Oct minutes](https://www.w3.org/2021/10/08-eo-minutes#item07). For example:
* Worried about the word deficiency being problematic, negative connotation.
* I feel like 'deficiency' suggests something that needs to be fixed

**update/clarification:** EOWG was fine with "color vision deficiency" when describing the medical condition -- just not about specific people or personas.

**Comment by mraccess77** [comment]
Hi Shawn, I can understand that concern with deficiency.   Color limited is my next recommendation.

**Reply by Myndex** [reply]
> Hi Shawn, I can understand that concern with deficiency. Color limited is my next recommendation.

Hi Jonathan @mraccess77  Well, **color limited** was my first choice as you know. I've been thinking about this for nearly two years, and that is the best I could find that satisfied all the conditions we've discussed.

**Reply by iadawn** [reply]
My concern on "colour limited" would be that it seems to run into the same problem as "colour blind" - it doesn't really bring any more accuracy or clarity to what is being described. It may well be that there isn't a pithy term that does do this but trying to build adoption of a new term would go much easier if the term did move things forward a bit.

**Reply by shawna-slh** [reply]
Ah, sorry that I was not clear enough in conveying my understanding of EOWG's perspectives. EOWG uncomfortable with "deficiency" **_only_** when describing a person or persona. They were fine with it when describing the medical condition. And they found that "color limited" was not clear. per: [minutes 22 Oct](https://www.w3.org/2021/10/22-eo-minutes#item04) and [minutes 29 Oct](https://www.w3.org/2021/10/29-eo-minutes#item02)

**Reply by mraccess77** [reply]
So it sounds like CVD is ok with EO for the condition - but in terms of how we refer to people we have to figure out something or just avoid using a term.

**Reply by shawna-slh** [reply]
In most current W3C resources, EOWG suggested not using a term at all. EOWG was comfortable with:
> Persona-name cannot distinguish between certain colors (often called “color blindness”).

That wording is shown in context in a [comment above](https://github.com/w3c/low-vision-a11y-tf/discussions/121).

While that uses a few more words, it is clear and understandable. And in W3C resources (except the one where I think everyone is comfortable using 'color vision deficiency), we are not repeating that phrase so it's OK that it is a few more words.

**Comment by mitchellevan** [comment]
I've always described myself as partially red-green colorblind. In a US context, I don't recall anybody expressing discomfort around the term _color blindness_, either from other colorblind people or from people with normal color vision. People do often ask me to explain what it is that I can or cannot see.

My initial reaction to this thread was that it's trying too hard to fix something that's not broken. However, this was partially because I didn't recall ever hearing the term _color vision deficiency_ before. Now that I've spent a few minutes skimming articles that use the term, I'm already getting used to it. I agree the term is already in common enough use in technical writing contexts, and it seems likely to reduce confusion. So I'm giving my +1 to _color vision deficiency_ as the preferred term in the W3C context.

That said, we shouldn't make extreme efforts to purge the term _color blindness_. It's widely recognized and not offensive in my experience (sample of one).

**Reply by shawna-slh** [reply]
Thank you so much for sharing your valuable perspective of one ! :-)

**Comment by shawna-slh** [comment]
Simpler language version of:
> cannot distinguish between certain colors (often called “color blindness”)

is:
> cannot see certain colors (often called “color blindness”)

or even simpler:
> cannot see some colors (often called “color blindness”)

**Questions:** How accurate is that? Even if not exactly technically/biologically/medically accurate, is it OK for personas that are not needing to be exactly accurate? Or, might it cause more confusion?

**Comment by mraccess77** [comment]
I don't think "cannot see certain colors (often called “color blindness”)
or even simpler: cannot see some colors (often called “color blindness”)" is accurate - or could be misunderstood.   They are likely to be differently than how other people may see them or may be hard to distinguish with other colors.

**Comment by Myndex** [comment]
Somewhat anecdotally, among the reasons I brought this forward as a useful task was in interviews with some test participants with CVD, some indicated they felt a negative stigma regarding the term "color blind", thus it seemed useful to explore alternatives.


## CVD By The Numbers

### Contents of this post moved to the[ LVTF CVD Wiki ](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/CVD)
This has multiple examples demonstrating how CVD sees the world.

**Comment by Myndex** [comment]
A comment I saw:

> _I'm **red-green insensitive,** and in reality that's exactly how it manifests, not as the inability to distinguish red from green. It's what you'd expect: Purples look more blue, greens look more yellow. That's (mostly) it._

I had not heard it described like that before, so:

## _Color Insensitive_

What do you think? I like it better than _Color Limited_

**Comment by shawna-slh** [comment]
[LVTF 18 November 2021 resolution to accept the replacement for colorblind in W3C WAI resources](https://www.w3.org/2021/11/18-lvtf-minutes.html#item01)

**Comment by vavroom** [comment]
Welp! Joining in late to the discussion because I only became aware of this when I read the blog post. Obviously y'all's decision to make, but... Frankly? You're splitting hairs. 

Some of you know me well enough to know I'm a strong advocate against ableist language. But this idea of replacing instances of "color blind" with "cannot distinguish between certain colors (often called "color blindness")" is verging on ludicrous.

Is there a lack of awareness around what "color blind" is? Absolutely. But you don't build awareness by using language that verges on political correctness for the sake of being PC. There are better ways to handle this. As someone said on Twitter, few people understand what a billion is. We don't start talking about "1 followed by 9 zeros"... We shouldn't do the same for color blindness. Or dysgraphia, or any numbers of other conditions that need further defining.

The very real risk here is to complicate language and readability of documents that are, already, hard to read and understand. You're going away from using Plain Language. And isn't WCAG being hard to read and understand one of the number 1 complaint against WCAG? Certainly is feedback I get a LOT from clients I work with.

Maybe it's too late, but I would STRONGLY urge you to:

1. Revisit the issue
2. Do some polling of actually color blind people. My sample size is limited, but I know at least 6 color blind folks who disagree with your approach... We tell clients to run usability studies with real disabled folks before launching a product. Maybe we should do the same here.

If continuing on this path, you could maybe use a hyperlink to a definition/explanation of what color blind is on the first instance of the term on a page.

**Reply by Myndex** [reply]
Hi @vavroom  Thanks for you comment. To be clear, the initial proposal was simply to use "color limited" instead of color blind. From my point of view this is not so much to be "politically correct" (which is something I personally detest) and instead be more descriptive. More recently, "color insensitive" seems like a good choice as well, it was suggested by a CVD individual. I missed the meeting when the resolution was made.

The motivation for this started a few years ago when I was developing [this clinically accurate CVD simulator](http://www.myndex.com/CVD/) and it became clear to me that "color blind" was inaccurate at best, but in that inaccuracy leads to much misunderstanding about the condition and how best to accommodate it. 

Existing standards (and not just here) indicate misconceptions on how best to accommodate CVD — the WCAG 2 contrast spec for instance makes the claim that it _"helps CVD"_ and _"the contrast is calculated in such a way that color is not a key factor"_ except that I've shown that WCAG2 contrast actually [makes things worse](https://gist.github.com/Myndex/c30dba273aa5eca426ad9f5200917c9d) in some cases. And this continues with odd advice regarding inline text links in the "understanding" docs which also provide no visual exampled... except where W3's CSS fails itself:

### Standard vision is upper left.
<img width="600" alt="Screen Shot 2021-12-04 at 5 54 43 AM" src="https://user-images.githubusercontent.com/42009457/144712134-1596755d-b806-4b58-8129-270faca49dad.png">

So in this snippet, information is coded in non-accessible colors for deutan and protan.

Interestingly, the WCAG 2 document rarely uses "color blind" instead using various terms such as "color deficit".

**_The upshot:_** My entire purpose here is to fix these issues of misunderstanding, at least for WCAG 3. When I brought up the subject, it was to find a different but still simple term, as I feel that "color vision deficiency" is a mouthful and feels a bit clumsy in writing.

I'm probably going to use either "color limited" or "color insensitive".


Thank you

Andy

**Reply by vavroom** [reply]
@Myndex That's fine - and indicates a need for education as well as change in many different ways. But the solution that has been adopted also makes it darn hard to read and understand what the thing is supposed to be about. Let's not gain accuracy by decreasing ease of understanding.

**Comment by mitchellevan** [comment]
I'm just tuning back in here, seeing that the Low Vision Task Force has moved away from a proposal of _color vision deficiency_ and adopted a [resolution](https://www.w3.org/2021/11/18-lvtf-minutes.html#item01) for _cannot distinguish between certain colors_ in most cases.

As a partially color-blind person, the latter term would not bother me. It describes my lived experience accurately, which was the original stated goal here.

On the other hand, a better measure of success of such a change would be: does it make things clearer and better for a wide audience of web authors? If a proposed change confuses as many people as it helps, then it's not worth making it. So, in addition to polling color-blind people as @vavroom suggested, it would also be very helpful to poll beyond our group of accessibility insiders.

> cannot distinguish between certain colors (often called “color blindness”)

In this context the quotation marks around the term _color blindness_ look like [scare quotes](https://www.merriam-webster.com/dictionary/scare%20quotes), incorrectly suggesting that the term is offensive or outdated. [Wikipedia style](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Text_formatting#Words_as_words) would use italics for naming a technical term, but [WAI style](https://www.w3.org/WAI/EO/wiki/Style) seems only to use quotation marks for this purpose, so I'm not sure what to do about this small ambiguity.

**Reply by Myndex** [reply]
Hi @mitchellevan  See also my reply to @vavroom  but in addition:

> I'm just tuning back in here, seeing that the Low Vision Task Force has moved away from a proposal of _color vision deficiency_ 

### How do you feel about the term _"color insensitive"_ ?

Thank you

Andy

**Comment by vavroom** [comment]
I'm still really unsure as to the need to change language at all. 

No dispute that "color blind" may be misunderstood by some people. Again this is a case in favor of education rather than wholesale language changes. 

"Color insensitive" isn't as much of a mouthful as "cannot distinguish between certain colors (often called "color blindness")". But it also invokes a negative trait - if you're insensitive, it's not great.

The [National Health Institute](https://www.nei.nih.gov/learn-about-eye-health/eye-conditions-and-diseases/color-blindness) uses the term and defines it. So does the [American Institute of Ophtalmology](https://www.aao.org/eye-health/diseases/what-is-color-blindness). The term color blind is the correct term for the condition. 

Mind you - I'm all for stepping away from the medical model and that may involve going against accepted conditions' names. Just still not convinced this is the case here.

But if the decision is to retain the expression, I would strongly urge the group to use it in the first instance on a page, and then to use one that is easier to read throughout the page. Even if you end up automatically linking to the definition on each instance of the word - I'm not sure that's necessary, but maybe it is.

**Reply by Myndex** [reply]
Hi Nicolas @vavroom 

> I'm still really unsure as to the need to change language at all.

I am suggesting it based on conversations with CVD test subjects.


> No dispute that "color blind" may be misunderstood by some people. Again this is a case in favor of education rather than wholesale language changes.

Language colors the perception. (see what I did there?)


> "Color insensitive" isn't as much of a mouthful as "cannot distinguish between certain colors (often called "color blindness")". But it also invokes a negative trait - if you're insensitive, it's not great.

I mentioned color insensitive as that is the term someone with CVD preferred.


> The National Health Institute

This is a body I plan to lobby for change, but wanted to discuss here first.


> The term color blind is the correct term for the condition.

In clinical studies, Color Vision Deficiency is the term most common used, and stated as CVD. But clinical papers often read like alphabet soup, and neither "CVD" nor "Color Vision Deficiency" are plain language friendly.

I am criticized for writing with too much jargon or too lengthy a manner. So I am looking for accurate but plain language substitutes for the clinical or scientific term in most of the work I have been doing.


The principal problem with blind is the interpretation of "nothing" and color blind is easily interpreted as "black & white" and while that does exist, it is both extremely rare, and most typically co-morbid with low vision and severe photophobia, requiring AT to some degree. In fact I had read some opinion letters that achromatopsia should not be grouped with other CVD types at all, in part do to the more serious co-morbid factors.

The vast majority of CVD see color, but varying degrees of limitations to the color.

But here's another factor, and it's part of the psychological aspect of the understanding - speaking of the two most common (deutan and protan) they are often said to be "red/green color blind, unable to distinguish red and green".

Okay, so take a look at these examples. Top left is standard vision. on the right deutan and protan:
<img width="420" alt="Screen Shot 2021-11-02 at 12 58 52 PM" src="https://user-images.githubusercontent.com/42009457/146628126-d31150b1-2ef7-4d33-95eb-bc1386c5fb16.png">

HMMMMMMMM....!!!

What's going on: red has a far lower luminance coefficient than green, so max green will be much lighter, even brighter, than max red by about 2 ½ times.

And notice also that, especially for protan, they are unable to distinguish green and YELLOW!

This is because yellow stimulates both the L and M cone about equally in normal vision ... but as red makes up so little of the luminance, someone with no red sensitivity (protan) essentially sees only the green, both for the plain rectangle and the yellow.

Protan, though the are red deficient or red-blind can "learn" to distinguish reds, they do so judging the relative luminance.

_TANGENT_
**Tragic**: But this had tragic consequences in aviation, there is a condition with PAPI approach lights at night for landings. There are 4 lights in a row, either white or red, depending on the angle of approach. For a correct glideslope, it is two white and two red. The accident pilot was protan, and had learned to judge by luminance. It was a "black hole" approach due to the weather conditions, with nothing but the PAPI to guide... but the aircraft was too low, showing four red lights. Because they were all the same and do to the upslope of that runway, the pilot though he was too high with 4 white lights, as there was no appreciable luminance difference between the four red lights. Had there been at least one white light, he likely would have recognized the luminance difference.

### _great Andy, get to the point already LOL_

As I am thinking about this now, _in a way_ the more accurate terms are "green blind" for deuteranopia, "red blind" for protanopia, and "blue blind" for tritanopia. But with the much more common anomalous forms, deuteranomalous -> green blind is incorrect, it would be reduced green sensitivity.

I'm not sure it answers the question regarding using a simple name, but what I have hoped to do is indicate that education and outreach about the understanding of CVD needs to be revisited. YES, at equal luminance, hue sensitivity is deeply impaired, but other things being equal a hue shift is accompanied by a luminance shift.

This is an important factor in considering design for CVD, and also common pitfalls, and stressing the need for accurate perceived lightness contrast.

Andy
